File: src/scanners/DolphinFlatpakSource.js
const DolphinSource = require("./DolphinSource.js");
const DolphinFlatpakGame = require("../games/DolphinFlatpakGame.js");

const USER_DIR = process.env["HOME"];

class DolphinFlatpakSource extends DolphinSource{

	static name = "Dolphin (Flatpak)";
	static gameClass = DolphinFlatpakGame;

	configPath = `${USER_DIR}/.var/app/org.DolphinEmu.dolphin-emu/config/dolphin-emu/Dolphin.ini`;

}

module.exports = DolphinFlatpakSource;

File: gali/sources/dolphin/dolphin_flatpak_source.py
from gali.sources.dolphin.dolphin_native_source import DolphinNativeSource
from gali.sources.dolphin.dolphin_flatpak_game import DolphinFlatpakGame
from gali.utils.locations import HOME


class DolphinFlatpakSource(DolphinNativeSource):

    name: str = "Dolphin (Flatpak)"
    game_class: type[DolphinFlatpakGame] = DolphinFlatpakGame
    config_path: str = f"{HOME}/.var/app/org.DolphinEmu.dolphin-emu/config/dolphin-emu/Dolphin.ini"